src/core/webdavconnection.h

```diff
@@ -87,7 +87,7 @@ class WebdavConnection : public QObject
 
     Q_INVOKABLE void importPath( const QString &remotePath, const QString &localPath );
     Q_INVOKABLE void downloadPath( const QString &localPath );
-    Q_INVOKABLE void uploadPath( const QString &localPath );
+    Q_INVOKABLE void uploadPaths( const QStringList &localPaths );
```

Comment:
Note to self: add documentation! :wink: That's on me, I'll take care of it after we merge this.